CMakeLists.txt

```diff
@@ -5,110 +5,115 @@ include(cmake/GitHelpers.cmake)
 
 git_description(GIT_DESCRIPTOR GIT_TAG GIT_COUNT)
 
-message(WARNING "TODO: Installing developer components has not been tested.")
+project(CVSLogger
+  VERSION 3.0.${GIT_COUNT}
+  DESCRIPTION "CVS logger library (${REV_ID})"
+  HOMEPAGE_URL "https://comvisionsys.ru")
 
-project(cvslogger
-    VERSION 2.0.${GIT_COUNT}
-    DESCRIPTION "CVS logger library (${REV_ID})")
+string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWER)
 
 option(CVSLOGGER_SHARED "" ON)
 option(CVSLOGGER_TESTS "" OFF)
 option(CVSLOGGER_OPENCV_IMG "" OFF)
 
-option(CVSLOGGER_INSTALL "" OFF)
-option(CVSLOGGER_DEV_INSTALL "" OFF)
-
 option(CVSLOGGER_ENABLE_STD_BY_DEFAULT "Enable stdout sink by default" OFF)
 option(CVSLOGGER_ENABLE_SYSD_BY_DEFAULT "Enable systemd sink by default" ON)
 
 include(GenerateExportHeader)
 
 if (NOT SPDLOG_FMT_EXTERNAL OR NOT SPDLOG_FMT_EXTERNAL_HO)
-    message(NOTICE "Set SPDLOG_FMT_EXTERNAL to ON. "
-        "FPSLogger is not compatible with internal fmt in spdlog version 1.8.2 and older.")
-    set(SPDLOG_FMT_EXTERNAL ON CACHE BOOL "")
+  message(NOTICE "Set SPDLOG_FMT_EXTERNAL to ON. "
+    "FPSLogger is not compatible with internal fmt in spdlog version 1.8.2 and older.")
+  set(SPDLOG_FMT_EXTERNAL ON CACHE BOOL "")
 endif()
 
 if(NOT TARGET fmt::fmt)
-    find_package(fmt REQUIRED)
+  find_package(fmt REQUIRED)
 endif()
 
-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-add_subdirectory(thirdparty/spdlog)
-
+find_package(spdlog)
 find_package(PkgConfig)
-pkg_check_modules(PC_SYSTEMD QUIET libsystemd)
+pkg_check_modules(systemd QUIET IMPORTED_TARGET libsystemd)
```

Comment:
why QUIET instead of REQUIRED? Older version with QUIET fails to compile exactly because of missing systemd on one clean system